src/sparse/impl/KokkosSparse_spmv_impl.hpp

```diff
@@ -567,315 +1116,83 @@ struct SPMV_MV_LayoutLeft_Functor {
 #pragma unroll
 #endif
 #ifdef KOKKOS_ENABLE_PRAGMA_LOOPCOUNT
-#pragma loop count (15)
+#pragma loop count(15)
```

Comment:
Do we still need this?  The point of this was to turn off vectorization on KNC, no?